lms/static/js/certificates/collections/certificate_whitelist.js

```diff
@@ -0,0 +1,93 @@
+// Backbone.js Application Collection: CertificateWhiteList
+/*global define, RequireJS */
+
+;(function(define){
+    'use strict';
+    define([
+            'backbone',
+            'gettext',
+            'js/certificates/models/certificate_exception'
+        ],
+
+        function(Backbone, gettext, CertificateExceptionModel){
+
+            var CertificateWhiteList =  Backbone.Collection.extend({
+                model: CertificateExceptionModel,
+
+                initialize: function(attrs, options){
+                    this.url = options.url;
+                },
+
+                getModel: function(attrs){
+                    var model = this.findWhere({user_name: attrs.user_name});
+                    if(attrs.user_name && model){
+                        return model;
+                    }
+                    else if(attrs.user_email){
+                        return this.findWhere({user_email: attrs.user_email});
+                    }
+                    else{
+                        return undefined;
+                    }
+                },
+
+                sync: function(options, appended_url){
+                    var filtered = this.filter(function(model){
+                        return model.isNew();
+                    });
+
+                    Backbone.sync(
+                        'create',
+                        new CertificateWhiteList(filtered, {url: this.url + appended_url}),
+                        options
+                    );
+                },
+
+                parse: function (certificate_white_list_json) {
```

Comment:
@saleem-latif I'm wondering why you need to override `parse` method of collection here?